compiler/luci/pass/src/FuseInstanceNormPass.cpp

```diff
@@ -730,6 +731,8 @@ template <> bool InstanceNormPattern::match<InstanceNormPattern::PatternVersion:
   CHECK_OR_FALSE(
     const_as_beta->dim(0).value() == 1 && const_as_beta->dim(1).value() == input_channel &&
     (const_as_beta->dim(2).value() == 1 || const_as_beta->dim(2).value() == input_last_dim));
+  // make clone for shared beta node that gets reshaped in reshape_gamma_beta()
+  const_as_beta = luci::clone(const_as_beta);
```

Comment:
The node can be cloned even when the pattern is not matched. How about doing this after all checks are done?